jenkins.yml

```diff
@@ -22,6 +22,22 @@
     sudo_defaults:
       - defaults: '!requiretty'
 
+  pre_tasks:
+    - name: Add jenkins service unit to systemd in case of docker.
+      copy:
+        dest: /etc/systemd/system/jenkins.service
+        content: |
```

Comment:
Instead of dropping the content directly in here, could you create a template or file and then deploy using the template or file module? You could probably use a template here, and fill in some of the paths with some of the variables provided by the Jenkins role.